reflex/components/component.py

```diff
@@ -1350,6 +1350,9 @@ def _get_imports(self) -> ParsedImportDict:
         # Get static imports required for event processing.
         event_imports = Imports.EVENTS if self.event_triggers else {}
 
+        # Get static imports required for error handling.
+        errors_imports = Imports.FRONTEND_ERRORS if self.tag == "ErrorBoundary" else {}
```

Comment:
these imports should just be added to the ErrorBoundary component instead of special casing them here